webapp/application.py

```diff
@@ -1,13 +1,25 @@
+import json
 import os
+from smtplib import SMTP
 import socket
-from dateutil.parser import parse
-
+from email.message import EmailMessage
+from email.utils import parseaddr
 import flask
 import talisker.requests
+from dateutil.parser import parse
 
 from webapp.greenhouse import Harvest
 from webapp.utils.cipher import Cipher
 
+
+withdrawal_reasons = {
+    "": "No reason given",
```

Comment:
@marinacastejon is the withdrawal reason optional?

Reply:
no, both the email address and the withdrawal reason are mandatory @minkyngkm Thanks